Sources/Logger/LoggerManager.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  LoggerManager.swift
+//  
+//
+//  Created by Martin Troup on 24.09.2021.
+//
+
+import Combine
+import Foundation
+
+public typealias ApplicationCallbackBundle = (callbacks: [ApplicationCallbackType], level: Level)
+public typealias MetaInformationBundle = (types: [MetaInformationType], bundle: Bundle, level: Level)
+
+/// LoggerManager manages different types of loggers. The class enables to register custom or pre-built loggers.
+/// Each of these logger classes must be subclassed from BaseLogger. The class handles logging to registered loggers
+/// based on levels they are set to acccept.
+public class LoggerManager {
+    // Configuration of logging mode
+    let loggingConcurrencyMode: LoggingConcurrencyMode
```

Comment:
Asi by mohlo být private?

Reply:
Ja to pouzivam v testech... Ono to stejne zvenci neni videt pre to neni public.